src/main/java/uk/gov/companieshouse/web/accounts/security/WebSecurityConfigurer.java

```diff
@@ -15,28 +16,30 @@ public class WebSecurityConfigurer {
 
     @Configuration
     @Order(1)
-    public static class GovUkAccountsSecurityFilterConfig extends WebSecurityConfigurerAdapter {
+    public static class GovUkAccountsSecurityFilterConfig {
 
-        @Override
-        protected void configure(HttpSecurity http)
-                throws Exception {
+        @Bean
+        public SecurityFilterChain govUkAccountsSecurityFilterChain(HttpSecurity http) throws Exception {
+            http
+                .securityMatcher("/accounts/**")
```

Comment:
Think this should be a requestMatcher to match previous behaviour, unless the intention is to exclusively apply security to /accounts/* endpoints for some reason

Reply:
Good spot, will do

Reply:
Done